mqtt/src/main/scala/akka/stream/alpakka/mqtt/Mqtt.scala

```diff
@@ -79,26 +82,67 @@ final case class MqttConnectionSettings(
     auth: Option[(String, String)] = None,
     socketFactory: Option[SSLSocketFactory] = None,
     cleanSession: Boolean = true,
-    will: Option[MqttMessage] = None
+    will: Option[MqttMessage] = None,
+    automaticReconnect: Boolean = false,
+    keepAliveInterval: FiniteDuration = 60.seconds,
+    connectionTimeout: FiniteDuration = 30.seconds,
+    maxInFlight: Int = 10,
+    mqttVersion: Int = MqttConnectOptions.MQTT_VERSION_3_1_1,
+    serverUris: Option[Seq[String]] = None,
```

Comment:
I don't see any reason to wrap the Seq in an Option. An empty sequence seems enough.